src/labels/CustomLabels.labels

```diff
@@ -1791,6 +1791,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>MetaDeploy link label</shortDescription>
         <value>Launch MetaDeploy</value>
     </labels>
+    <labels>
```

Comment:
![Screen Shot 2020-04-15 at 3 12 22 PM](https://user-images.githubusercontent.com/57271120/79394234-d9a4d100-7f2b-11ea-9b7f-fb861c6e3db8.png)
@sam-knox Please review the new labels.

Reply:
@andrewyu-salesforce - The label looks fine, but is it possible to bold the line starting with "Because you have no Recurring Donations . . . ". It just feels a little lost in that paragraph.

Reply:
![Screen Shot 2020-04-15 at 3 47 01 PM](https://user-images.githubusercontent.com/57271120/79396718-643bff00-7f31-11ea-9c7a-2a773dad34a0.png)
@sam-knox done

Reply:
@andrewyu-salesforce - Awesome, thanks. I noticed one more change needed, though. "Recurring Donations" should be singular in that sentence so it reads:

"Because you have no Recurring Donation records . . . "

Reply:
![Screen Shot 2020-04-16 at 11 05 07 AM](https://user-images.githubusercontent.com/57271120/79490722-2cd15f00-7fd2-11ea-88b9-3e919c802116.png)
@sam-knox Thank you for continually going through the text. The 's' is removed

Reply:
Thanks @andrewyu-salesforce!